src/main/java/org/candlepin/subscriptions/db/model/EventKey.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.db.model;
+
+import org.candlepin.subscriptions.json.Event;
+
+import lombok.Getter;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+/**
+ * Defines the unique constraints for an EventRecord. This object is primarily used for creating
+ * lookup tables while processing Events pulled from the DB.
+ */
+@Getter
+public class EventKey {
+
+    private String accountNumber;
+    private String eventType;
+    private String eventSource;
+    private String instanceId;
+    private OffsetDateTime timestamp;
+
+    public EventKey(String accountNumber, String eventType, String eventSource, String instanceId,
```

Comment:
Can you please swap the order to be `eventSource`, then `eventType`, to be consistent with the repository method?

Reply:
DONE